platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -102,6 +109,32 @@
 
   private String foreground;
 
+  public void setSourceKeyword(ResourceBundleLocator resourceBundleLocator) {
+    try {
+      ResourceBundle resourceBundle = resourceBundleLocator.getBundle(LANDING_PAGE_BASE_NAME);
+
+      if (resourceBundle.containsKey("dataSource")) {
+        this.sourceKeyword = (String) resourceBundle.getObject("dataSource");
+      } else {
+        this.sourceKeyword = "Data Source";
+      }
+    } catch (IOException e) {
+      LOGGER.debug(
+          "An error occurred while creating class loader to URL for ResourceBundle: {}, {}",
+          LANDING_PAGE_BASE_NAME,
+          Locale.getDefault(),
+          e);
+    }
+  }
+
+  public void setSourceKeyword(String sourceKeyword) {
```

Comment:
❗️ Same comment as before. If these 2 methods are for testing than make them package private and annotate them with `@VisibleForTesting`

Reply:
I am using them for testing purposes and blueprint was throwing an error for not having a corresponding getter for a setter. I've made them package private and added @VisibleForTesting